### DISCUSSION TITLE
For Exporting BIRT reports in Excel is there way use .xlsx besides .xls

### DISCUSSION THREAD

**Discussion by tnation1960** [eclipse-birt/birt#1030] [General]
When Exporting BIRT reports in Excel is there way use .xlsx besides .xls

**Comment by tnation1960** [comment]
Little more information to my question: The client view Microsoft File Types:  these are older Microsoft Office files created prior to Office 2016 (having a .doc, .xls, or .ppt extension), and are now blocked from opening; this is not related to the O365 update, but is due to network security compliance. BIRT reports exported to .xls the users are not able to open.  So, is there way for exporting Excel using .xlsx instead of .xls.
Thank you

**Comment by hvbtup** [comment]
The short answer is yes: I'm pretty sure that the Spudsoft Emitter supports this.

Only I don't know if there is anything you have to configure to use it...

**Comment by tnation1960** [comment]
First, thank you for responding to my question!  I have another question. The problem I'm facing is in the BIRT Export window that .xlsx is not an option. What is the procedure to make show? 



    On Thursday, August 18, 2022 at 06:42:25 AM PDT, Henning von Bargen ***@***.***> wrote:  
 
 


The short answer is yes: I'm pretty sure that the Spudsoft Emitter supports this.

Only I don't know if there is anything you have to configure to use it...

**Comment by hvbtup** [comment]
It seems that you are using a very old BIRT release.

**Comment by TheLaserNinja** [comment]
I believe we are currently using some sort of binary encoded XLSX. We just kept the extension as XLS in earlier versions when writing out for whatever reason. IIRC, it had something to do with the XLS extension having a specific handler in the core engine for use with the commercial iHub native Excel emitter. But this would have been back in the 3.7 and maybe early version 4.X days. Sounds like you are using an older version.
